main/xf86-input-libinput/APKBUILD

```diff
@@ -1,17 +1,18 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=xf86-input-libinput
-pkgver=0.26.0
+pkgver=0.27.0
 pkgrel=0
 pkgdesc="X.Org input driver based on libinput"
-url="http://xorg.freedesktop.org/"
+url="http://xorg.freedesktop.org"
 arch="all"
 license="MIT"
 depends=""
 depends_dev="libinput-dev xorg-server-dev"
 makedepends="$depends_dev resourceproto scrnsaverproto eudev-dev"
 subpackages="$pkgname-dev $pkgname-doc"
-source="http://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2"
+source="https://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2
+	fix-apply-left-handed-property.patch"
```

Comment:
Could you explain why this patch is necessary? If it has been taken from upstream please include a git-format-patch instead.

Reply:
Upstream patch amended.